app/Http/Services/CustomErrorService.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Services;
+
+use Illuminate\Http\Response;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class CustomErrorService
+{
+    /**
+     * @param      $message
+     * @param      $errors
+     * @param  int $status
+     * @return \Symfony\Component\HttpFoundation\JsonResponse
+     */
+    public static function failedApiResponse($message, $errors, $status = Response::HTTP_BAD_REQUEST): JsonResponse
```

Comment:
Dit PR gaat om deze functie. Ik wil de error-messages beter uitlezen en plaatsen in het model. Daarom geeft deze functie dezelfde response terug als wanneer er server-errors ontstaan. Hierdoor kan ik specifieke errors bij specifieke properties van models plaatsen.